src/Workspaces/Core/Desktop/Workspace/SQLite/SQLitePersistentStorageService.cs

```diff
@@ -64,6 +64,13 @@ protected override string GetDatabaseFilePath(string workingFolderPath)
                 sqlStorage.Initialize(solution);
 
             }
+            catch (Exception ex) when (ex is DllNotFoundException || ex is EntryPointNotFoundException)
```

Comment:
Need to catch TypeInitializationException and check InnerException for the data and throw that